CHANGELOG.md

```diff
@@ -9,10 +9,11 @@ Dates in this file are in [Holocene Calendar] because it is amazing, logical, an
 ## [Unreleased]
 
 ### Added
-  - Orb radar can now show Orb Room both in NG and NG+ (Checkbox at the bottom of the radar).
+  - Orb radar can now show Orb Room both in NG and NG+ (Checkbox at the bottom of the radar). @Larandar
+  - Orb radar now filter collected Orbs (i.e. current PW chest orbs, or Orb Room). @Laradar
 
 ### Changed
-- Searching algorithm now work in a spiral pattern around the player.
+  - Searching algorithm now work in a spiral pattern around the player. @Larandar
```

Comment:
`work` -> `processes chunks`

Add a second line
`- Searching algorithm now works on noita chunks instead of custom internal 1024x1024 chunks to avoid confusion. @Larandar`
too I guess

Reply:
Done